docs/samples/git-setup/clustertemplates/hardwaretemplates/sno-ran-du/placeholder-du-template-v1.yaml

```diff
@@ -6,12 +6,14 @@ metadata:
 spec:
   # hardwareProvisioningTimeout is optional.
   # The value should be a duration string
-  # (e.g., "30m" for 30 minutes)
-  # hardwareProvisioningTimeout: "30m"
-  hardwarePluginRef: oran-hwmgr-plugin-test
+  # (e.g., "90m" for 90 minutes)
+  # hardwareProvisioningTimeout: "90m"
+  hardwarePluginRef: metal3-hwplugin
   bootInterfaceLabel: bootable-interface
   nodeGroupData:
     - name: controller
       role: master
-      hwProfile: profile-proliant-gen11-dual-processor-256G-v1
-      resourcePoolId: master-pool
+      hwProfile: profile-dell-r740-v1
+      resourceSelector:
```

Comment:
I think it would be nice to also include examples of the supported `hardwaredata` selectors (code [here](https://github.com/openshift-kni/oran-o2ims/blob/main/hwmgr-plugins/metal3/controller/resource_selection.go#L254-L259)):
```
   - "hardwaredata/cpu_arch"
   - "hardwaredata/cpu_model;substring;icase"
   - "hardwaredata/num_threads;gte"
   - "hardwaredata/nic;model=Intel;speedGbps>=10"
   - "hardwaredata/storage;type=SSD;count>=2"
```